src/apps/frontend/components/otp/otp-input.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FocusEventHandler } from 'react';
+
+type OTPInputProps = {
+  disabled: boolean;
+  error: string;
+  index: number;
+  inputRef: React.RefObject<HTMLInputElement[]> | null;
+  name: string;
+  onBlur?: FocusEventHandler<HTMLInputElement>;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onKeyDown: (event: React.KeyboardEvent<HTMLInputElement>) => void;
+  value: string;
+};
+
+const OTPInput: React.FC<OTPInputProps> = ({
+  disabled,
+  error,
+  index,
+  inputRef,
+  name,
+  onBlur,
+  onChange,
+  onKeyDown,
+  value,
+}) => (
+  <input
+    className={`
+      flex
+      w-full
+      rounded-lg
+      border
+      bg-transparent
+      py-4
+      text-center
+      outline-none
+      [appearance:textfield]
+      focus:border-primary
+      focus-visible:shadow-none
+      [&::-webkit-inner-spin-button]:appearance-none
+      ${error ? 'border-red-500' : 'border-stroke'}
+    `}
```

Comment:
Please create input component, and ensure it is used here and other places. I don't expect other pages to have any different treatment for input box.

Reply:
Does `maxlength =1` on input does not work?

Reply:
There are a few differences between the OtpInput and Normal Input that we have.
We want the text to be in the center, the onKeyDown function in the case of OtpInput.
If we can have a prop for textAlign: center, then it can be made reusable.

Reply:
Can the parent component not control the text alignment instead input?

Its ok to have input to accept keyPressHandler

Reply:
Input has been re-used for Otp Inputs, need your review on that.